pydcs_extensions/fa18efg/fa18efg.py

```diff
@@ -156,11 +255,8 @@ class Pylon1:
         AN_ASQ_T50_TCTS_Pod___ACMI_Pod = (1, Weapons.AN_ASQ_T50_TCTS_Pod___ACMI_Pod)
 
     class Pylon2:
-        TLAU_127 = (2, WeaponsFA18EFG.TLAU_127)
-        AIM_120B_AMRAAM___Active_Radar_AAM = (
-            2,
-            Weapons.AIM_120B_AMRAAM___Active_Radar_AAM,
-        )
+        TLAU_127 = (2, Weapons.TLAU_127)
```

Comment:
This weapon is actually defined in this file, so it _shouldn't_ reference pydcs' `Weapons` class. The idea is to keep pydcs clean of any mods, which I believe is still the case so this could cause trouble 😬

Can you start by reverting changes where `WeaponsF18EFG` was replaced with `Weapons`?

Reply:
My apologies, I recently finished support for the mod in liberation and, being aware that the version in retribution is outdated, copied over the files from my PR. Changes should be reverted in last push.

Reply:
You seem to have replaced _all_ occurrences, but it should only be applied to the weapons that are part of the mod, i.e. weapons that you defined at the start of the pydcs extension 😅

I recommend looking at one of the other mods to understand when to use `Weapons<Mod>` and when to use `Weapons`.

Bottom line, any stock DCS weapon should be referred to from the `Weapons` class while modded weapons use the custom defined weapons class in the pydcs extension.

Let me know if I can be of any further assistance 😉

Reply:
I believe the classes are correctly referred to now

Reply:
Just got it and properly changed it, thanks